src/net_processing.cpp

```diff
@@ -107,10 +107,10 @@ static const int MAX_BLOCKTXN_DEPTH = 10;
  *  degree of disordering of blocks on disk (which make reindexing and pruning harder). We'll probably
  *  want to make this a per-peer adaptive value at some point. */
 static const unsigned int BLOCK_DOWNLOAD_WINDOW = 1024;
-/** Block download timeout base, expressed in millionths of the block interval (i.e. 10 min) */
-static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 1000000;
-/** Additional block download timeout per parallel downloading peer (i.e. 5 min) */
-static const int64_t BLOCK_DOWNLOAD_TIMEOUT_PER_PEER = 500000;
+/** Block download timeout base, expressed in millionths of the block interval (i.e. 5 min) */
+static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 50000000;
```

Comment:
Based on your comment from line 110 - you wanted to reduce the timeout 2 times. But the value has been increased 10 times. One of these is wrong, please correct.

Reply:
Great catch! Thanks!

This should be `5000000`

Reply:
```suggestion
static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 5000000;
```